### PR TITLE
The IPFS Project Roadmap

### PR DESCRIPTION
**Pull request by daviddias**
## The IPFS Project Roadmap is here \o/

It is with great pleasure to announce on behalf of all the extraordinary humans that are part of the IPFS Org that we now have a full IPFS Project Roadmap!

image

This Roadmap will stay in review stage until the first week of January 2019, when we tick the version to v1.0.0 and do a broadcast introducing it to the whole World. Until then, all the feedback is very much appreciated, feel welcome to post comments on this thread directly! Thank you!

### CODE REVIEW

**Comment by daviddias** [comment]
@ipfs/working-group-captains can I have your review and approval to merge the Roadmap into master?

**Review by vasco-santos — APPROVED** [review]
👍 🚀

File: README.md
Comment (lidel) [inline README.md]: Before we merge either add the description or remove this line :)
```suggestion

```

**Review by lidel — APPROVED** [review]
:+1:

File: README.md
Comment (alanshaw) [inline README.md]: Missing #

File: README.md
Comment (alanshaw) [inline README.md]: Add "?" at the end?

File: README.md
Comment (alanshaw) [inline README.md]: Link not working

File: README.md
Comment (alanshaw) [inline README.md]: Suggestion: coalesced -> distilled. This isn't just all of them bundled together as one, these epics have extracted the essential meaning and most important aspects of the WG milestones.

File: README.md
Comment (alanshaw) [inline README.md]: become -> becomes

File: README.md
Comment (alanshaw) [inline README.md:100]: "Both teams have a chat" => "Both teams have a meet up"?

File: README.md
Comment (alanshaw) [inline README.md:104]: "Package Managers Registries" => "Package Manager's registries"

File: README.md
Comment (alanshaw) [inline README.md:105]: "understand each one challenges" => "understand each ones challenges"

File: README.md
Comment (alanshaw) [inline README.md:107]: "Package Managers users" => "Package Manager's users"

File: README.md
Comment (alanshaw) [inline README.md]: "We will booting the" => "We will be booting the"

Comment (alanshaw) [reply README.md]: "soon" => "in early 2019"?

File: README.md
Comment (alanshaw) [inline README.md:115]: hmm, I always figured this was more about us understanding how IPFS is growing. Opt-in sounds disabled by default (not what we want), but I think opt-out is not right as I'd like to think we'd confirm with people before just collecting and reporting them. Maybe "optional" is a better word?

File: README.md
Comment (alanshaw) [inline README.md]: Needs an explaination of what "DApp" stands for.

File: README.md
Comment (alanshaw) [inline README.md:123]: "Users" => "users"?

File: README.md
Comment (alanshaw) [inline README.md]: "Dapps" => "DApps"

File: README.md
Comment (alanshaw) [inline README.md]: Is "WebPlatform" meant to be one word?

...I didn't realise that was a thing...I have no idea what I'm doing dog <--me

File: README.md
Comment (alanshaw) [inline README.md]: "participate the" => "please come and participate in the"

File: README.md
Comment (alanshaw) [reply README.md]: Suggestion: make these headings so we can link to them.

File: README.md
Comment (alanshaw) [inline README.md]: "3000++" => "3,000+"?

File: README.md
Comment (alanshaw) [inline README.md]: Link to the org?

**Review by alanshaw — CHANGES_REQUESTED** [review]

**Review by alanshaw — APPROVED** [review]

File: README.md
Comment (olizilla) [inline README.md:48]:
```suggestion
- **🔄 Building the Decentralized Web**: IPFS wants to decentralize the web. The communities building the DWeb are already huge proponents of IPFS. By providing additional tools and capabilities we can support and amplify the movement. Augmenting the power and number of DWeb Applications created will increase the size and impact of our existing community, and test IPFS in ways only our users can imagine.
```

File: README.md
Comment (olizilla) [inline README.md:59]:
```suggestion
We designed a sorting function to help us identify goals that 1) we could focus on landing end-to-end in 2019, 2) would bring new partners and users into the community, and 3) would set us up for future goals while delivering near-term value.
```

File: README.md
Comment (olizilla) [inline README.md:67]: This list would benefit from an introduction. Right now the doc goes from high-level goals to a list of working group links, without explaining the link.

File: README.md
Comment (olizilla) [inline README.md:87]:
```suggestion
We've distilled the key themes from the Working Group Roadmaps into a list Epic Endevours that give an overview of the primary targets IPFS has for 2019. The details about the tasks and priorities that get us there are defined in the Working Group Roadmaps.
```

File: README.md
Comment (olizilla) [inline README.md:93]:
```suggestion
We've been working hard on improving and iterating on the IPFS Protocol to make sure it serves its users needs and achieves the goal of giving the Web a new super power: Content Addressing. In 2019 we will solidify the core components and ensure IPFS is ready for production systems. Our goals include:
```

File: README.md
Comment (olizilla) [inline README.md:97]: Link to specs here...
```suggestion
- Go and JS IPFS enable modern IPFS data formats ([UnixFSv2](https://github.com/ipfs/unixfs-v2), [CIDv1](https://github.com/ipld/cid), raw blocks) by default
```

File: README.md
Comment (olizilla) [inline README.md:100]:
```suggestion
You can contribute to these Epic by testing, debugging, documenting or coding on either Go or JS IPFS. Both teams have a chat every week that you can join. [Go](https://github.com/ipfs/team-mgmt/blob/master/MGMT_GOLANG_CORE_DEV.md) & [JS](https://github.com/ipfs/team-mgmt/blob/master/MGMT_JS_CORE_DEV.md#weekly-core-dev-team-calls).
```

**Review by olizilla — COMMENTED** [review]
I'm working on tightening up the copy. Am half-way through, but sharing my thoughts now, as the long document is long.

**Comment by olizilla** [comment]
I missed the boat with my feedback. I'll pull it into a PR.

File: README.md
Comment (daviddias) [reply README.md:67]: Good point! @olizilla would you like to volunteer and submit a PR to add that introduction?

**Comment by daviddias** [comment]
@olizilla thank you!

**Pull request merged**
